src/main/java/seedu/address/logic/commands/AddVolunteerCommand.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_BIRTHDAY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_GENDER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER_TAG;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.volunteer.Volunteer;
+
+/**
+ * Adds a volunteer to the address book.
+ */
+public class AddVolunteerCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a volunteer to the address book. "
+            + "Parameters: "
+            + PREFIX_VOLUNTEER_NAME + "NAME "
+            + PREFIX_VOLUNTEER_GENDER + "GENDER "
+            + PREFIX_VOLUNTEER_BIRTHDAY + "BIRTHDAY "
+            + PREFIX_VOLUNTEER_PHONE + "PHONE "
+            + PREFIX_VOLUNTEER_EMAIL + "EMAIL "
+            + PREFIX_VOLUNTEER_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_VOLUNTEER_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_VOLUNTEER_NAME + "John Doe "
+            + PREFIX_VOLUNTEER_GENDER + "male "
+            + PREFIX_VOLUNTEER_BIRTHDAY + "01-10-1985 "
+            + PREFIX_VOLUNTEER_PHONE + "98765432 "
+            + PREFIX_VOLUNTEER_EMAIL + "johnd@example.com "
+            + PREFIX_VOLUNTEER_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_VOLUNTEER_TAG + "friends "
+            + PREFIX_VOLUNTEER_TAG + "owesMoney";
```

Comment:
Might want to change the second tag to fit a volunteer's context instead.

Instead of 'owed money', a tag which says 'driver' would be better. (In the context of volunteer)